test is failing for new version of geopandas

`geopandas` version 0.6 now uses the `pandas` extension array system. The datatype for a `geometry` column in a `geopandas.GeoDataFrame` is now `geometry`, it used to be `object`. As a result, the test in `tests/test_file_source.py` is failing with the following error 

```
def test_shape_datasource(shape_datasource):
        info = shape_datasource.discover()
    
>       assert info['dtype'] == {'name': 'object',
                                 'marker-col': 'object',
                                 'marker-sym': 'object',
                                 'line': 'object',
                                 'geometry': 'object'}
E       AssertionError: assert {'geometry': ...'object', ...} == {'geometry': ...'object', ...}
E         Omitting 4 identical items, use -vv to show
E         Differing items:
E         {'geometry': 'geometry'} != {'geometry': 'object'}
E         Use -v to get the full diff

tests/test_file_source.py:35: AssertionError
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -20,25 +20,8 @@ script:
 - |
   flake8 .
   if [ "$TRAVIS_OS_NAME" = "linux" ]; then
-      conda build -c intake -c defaults -c conda-forge ./conda
+      conda build -c defaults -c conda-forge ./conda
   else
       # Workaround for Travis-CI bug #2: https://github.com/travis-ci/travis-ci/issues/7773
-      conda build -c intake -c defaults -c conda-forge --no-test ./conda
+      conda build -c defaults -c conda-forge --no-test ./conda
   fi
-- |
-  if [ -n "$TRAVIS_TAG" ]; then
-      # If tagged git version, upload package to main channel
-      anaconda -t ${ANACONDA_TOKEN} upload -u informaticslab --force `conda build --output ./conda`
-  elif ! [ -n "$TRAVIS_PULL_REQUEST" ]; then
-      # Otherwise upload package to dev channel
-      anaconda -t ${ANACONDA_TOKEN} upload -u informaticslab --label dev --force `conda build --output ./conda`
-  fi
-
-deploy:
-  provider: pypi
-  user: $PYPI_USER
-  password: $PYPI_PASSWORD
-  skip_cleanup: true
-  on:
-    tags: true
-    repo: informatics-lab/intake_geopandas

--- a/README.md
+++ b/README.md
@@ -1,8 +1,8 @@
 # intake_geopandas
 
-[![Build Status](https://travis-ci.com/informatics-lab/intake_geopandas.svg?branch=master)](https://travis-ci.com/informatics-lab/intake_geopandas)
+[![Build Status](https://travis-ci.com/intake/intake_geopandas.svg?branch=master)](https://travis-ci.com/intake/intake_geopandas)
 
-intake_geopandas: [Geopandas](http://geopandas.org/) plugin for [Intake](https://github.com/informatics-lab/intake_geopandas)
+intake_geopandas: [Geopandas](http://geopandas.org/) plugin for [Intake](https://github.com/intake/intake)
 
 See [Intake docs](https://intake.readthedocs.io/en/latest/overview.html).
 
@@ -15,8 +15,8 @@ It currently supports reading from the following data sources:
 
 ### Installation
 
-The conda install instructions are:
+The pip install instructions are:
 
 ```
-conda install -c informaticslab -c intake intake_geopandas
+pip install intake_geopandas
 ```

--- a/conda/meta.yaml
+++ b/conda/meta.yaml
@@ -32,8 +32,8 @@ test:
     - py.test --verbose
 
 about:
-  home: https://github.com/informaticslab/intake-geopandas
+  home: https://github.com/intake/intake-geopandas
   license: BSD
   license_file: LICENSE
   summary: |
-    Geopandas plugin for Intake
+    Geopandas plugin for Intake, providing drivers for Shapefiles, GeoJSON, PostGIS, and Spatialite

--- a/tests/test_file_source.py
+++ b/tests/test_file_source.py
@@ -2,43 +2,56 @@
 import os
 import pytest
 
+from pkg_resources import get_distribution, parse_version
+
 from intake_geopandas import GeoJSONSource, ShapefileSource
+
+geom_col_type = (
+    "object"
+    if parse_version(get_distribution("geopandas").version) < parse_version("0.6")
+    else "geometry"
+)
 
 
 @pytest.fixture
 def shape_filenames():
     basedir = os.path.dirname(__file__)
-    return dict(
-        stations=os.path.join(basedir, 'data', 'stations', 'stations.shp')
-    )
+    return dict(stations=os.path.join(basedir, "data", "stations", "stations.shp"))
 
 
 @pytest.fixture
 def shape_datasource(shape_filenames):
-    return ShapefileSource(shape_filenames['stations'])
+    return ShapefileSource(shape_filenames["stations"])
 
 
 @pytest.fixture
 def geojson_filenames():
     basedir = os.path.dirname(__file__)
-    return dict(countries=os.path.join(basedir, 'data', 'countries.geo.json'))
+    return dict(countries=os.path.join(basedir, "data", "countries.geo.json"))
 
 
 @pytest.fixture
 def geojson_datasource(geojson_filenames):
-    return GeoJSONSource(geojson_filenames['countries'])
+    return GeoJSONSource(geojson_filenames["countries"])
 
 
 def test_shape_datasource(shape_datasource):
     info = shape_datasource.discover()
-
-    assert info['dtype'] == {'name': 'object',
-                             'marker-col': 'object',
-                             'marker-sym': 'object',
-                             'line': 'object',
-                             'geometry': 'object'}
+    shape_datasource.read()
+    assert info["dtype"] == {
+        "name": "object",
+        "marker-col": "object",
+        "marker-sym": "object",
+        "line": "object",
+        "geometry": geom_col_type,
+    }
 
 
 def test_countries_datasource(geojson_datasource):
     info = geojson_datasource.discover()
-    print(info)
+    geojson_datasource.read()
+    assert info["dtype"] == {
+        "geometry": geom_col_type,
+        "id": "object",
+        "name": "object",
+    }

